Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -35,8 +35,8 @@ function FailBuildOnQualityGateStatus
     {        
         $dashboardUrl = GetTaskContextVariable "MSBuild.SonarQube.ProjectUri"
         
-        Write-Host "##vso[task.logissue type=error]The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
-        Write-Host "##vso[task.complete result=Failed;]"
+        Write-VstsTaskError "The SonarQube quality gate associated with this build has failed. For more details see $dashboardUrl"
```

Comment:
Oh OK, it may not make sense then for that task. I'll leave that to your team to figure out. If you need help with it, let me know.